### PR TITLE
nexous.de

### PR DESCRIPTION
**Pull request by Phyroks**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/nexousde/background.png
Comment (DoJapHD) [inline minecraft_servers/nexousde/background.png:1]: Compress file

File: minecraft_servers/nexousde/background@2x.png
Comment (DoJapHD) [inline minecraft_servers/nexousde/background@2x.png:1]: Compress file

File: minecraft_servers/nexousde/manifest.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "server_name": "nexousde",
+  "nice_name": "Nexous.DE",
+  "direct_ip": "nexous.de",
+  "supported_languages": [
+    "de",
+    "en"
+  ],
+  "social": {
+    "web": "https://nexous.de",
```
Comment (DoJapHD) [inline minecraft_servers/nexousde/manifest.json:10]: Website isn't really there (just redirects)

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by Phyroks** [comment]
@DoJapHD Images are now compressed :)

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**